run_bot.sh

```
@@ -246,31 +422,222 @@ clean_recordings() {
+    
+    # Construire l'image si nécessaire
+    if ! docker images | grep -q meet-teams-bot; then
+        print_info "Docker image not found, building..."
+        build_image
+    fi
+    
+    print_info "🚀 Starting normal bot run with screen recording..."
+    print_info "ℹ️ Will automatically stop after ${duration}s"
+    
+    # Créer un fichier temporaire pour les logs
+    local log_file="/tmp/test-run-$(date +%s).log"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)